frame/contracts/src/weights.rs

```diff
@@ -182,6 +183,9 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
+	fn on_idle() -> Weight {
+		Self::on_initialize()
+	}
```

Comment:
This file is auto generated when the benchmarks are run. So we can't make such changes here. Maybe just rename the benchmark to `process_deletion_queue_batch`. Then it is general enough to use it for both.

Reply:
Done